build.go

```diff
@@ -343,7 +343,7 @@ func setBuildEnv() {
 		os.Setenv("GOARCH", goarch)
 	}
 	if goarch == "386" {
-		os.Setenv("GO386", "387")
+		os.Setenv("GO386", "softfloat")
 	}
```

Comment:
Please drop this whole section, so we'll use the default instead ("sse2").

Reply:
OK, this statement is removed in [e091803](https://github.com/lelibreauquotidien/grafana/commit/e09180346ddbae24c17eaafa0220845567709a2b)